### PR TITLE
Add new

### PR DESCRIPTION
**Pull request by guadarickel**

### CODE REVIEW

**Review by peirano357 — APPROVED** [review]

**Review by peirano357 — CHANGES_REQUESTED** [review]
lang attribute should be "es". please check.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (peirano357) [inline index.html:2]: this should be in spanish

**Review by peirano357 — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (guadarickel) [reply index.html:2]: The site will be in english!

Comment (peirano357) [reply index.html:2]: you are right. My bad

**Pull request merged**